app/src/ui/lib/title-bar-style.ts

```diff
@@ -0,0 +1 @@
+export type TitleBarStyle = 'native' | 'custom'
```

Comment:
:art: let's document this enum value as well

```suggestion
/**
 * This string enum represents the supported modes for rendering the title
 * bar in the app.
 *
 *  - 'native' - use the default window style and chrome supported by the window
 *               manager
 *
 *  - 'custom' - hide the default window style and chrome and display the menu
 *               provided by GitHub Desktop
 *
 * This is only available on the Linux build. For other operating systems this
 * is not configurable:
 *
 *  - macOS uses the native title bar
 *  - Windows uses the custom title bar
 */
export type TitleBarStyle = 'native' | 'custom'
```